pkg/api/handlers/generic/images.go

```diff
@@ -0,0 +1,396 @@
+package generic
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/containers/libpod/pkg/api/handlers"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/containers/buildah"
+	"github.com/containers/image/v5/manifest"
+	"github.com/containers/libpod/libpod"
+	image2 "github.com/containers/libpod/libpod/image"
+	"github.com/containers/libpod/pkg/util"
+	"github.com/containers/storage"
+	"github.com/docker/docker/api/types"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+func saveFromBody(f *os.File, r *http.Request) error { // nolint
+	if _, err := io.Copy(f, r.Body); err != nil {
+		return err
+	}
+	return f.Close()
+}
+
+func LoadImage(w http.ResponseWriter, r *http.Request) {
+	runtime := r.Context().Value("runtime").(*libpod.Runtime)
+
+	var (
+		err    error
+		writer io.Writer
+	)
+	quiet := false
+	if len(r.Form.Get("quiet")) > 0 {
+		quiet, err = strconv.ParseBool(r.Form.Get("quiet"))
+		if err != nil {
+			handlers.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "bad value for quiet"))
+			return
+		}
+	}
+	f, err := ioutil.TempFile("", "api_load.tar")
+	if err != nil {
+		handlers.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "failed to create tempfile"))
```

Comment:
No capitalized errors.

Reply:
to be consistent with the other api, we must use caps.